crm_location/README.rst

```diff
@@ -0,0 +1,46 @@
+CRM location
```

Comment:
What happend? the file has the same structure to this https://raw.githubusercontent.com/OCA/maintainer-tools/master/template/module/README.rst